.github/workflows/build.yml

```diff
@@ -93,37 +90,42 @@ jobs:
       - name: Prepare runner
         uses: ./.github/actions/prepare_runner
         with:
-          disable_ccache: false
+          disable_ccache: ${{ inputs.disable_cache }}
 
       - name: Setup conan
         uses: ./.github/actions/setup_conan
         id: conan
         with:
-          conan_profile: ${{ matrix.conan_profile }}
+          conan_profile: ${{ inputs.conan_profile }}
 
       - name: Restore cache
+        if: ${{ !inputs.disable_cache }}
         uses: ./.github/actions/restore_cache
         id: restore_cache
         with:
           conan_dir: ${{ env.CONAN_USER_HOME }}/.conan
           conan_profile: ${{ steps.conan.outputs.conan_profile }}
           ccache_dir: ${{ env.CCACHE_DIR }}
-          build_type: ${{ matrix.build_type }}
-          code_coverage: ${{ matrix.code_coverage }}
+          build_type: ${{ inputs.build_type }}
+          code_coverage: ${{ inputs.code_coverage }}
 
       - name: Run conan and cmake
         uses: ./.github/actions/generate
         with:
           conan_profile: ${{ steps.conan.outputs.conan_profile }}
-          conan_cache_hit: ${{ steps.restore_cache.outputs.conan_cache_hit }}
-          build_type: ${{ matrix.build_type }}
-          code_coverage: ${{ matrix.code_coverage }}
-          static: ${{ matrix.static }}
+          conan_cache_hit: ${{ !inputs.restore_cache && steps.restore_cache.outputs.conan_cache_hit }}
```

Comment:
It seems there is no `inputs.restore_cache`.

Reply:
Good catch. I think i had them as separate inputs at first but then decided it could be covered by `conan_cache` but forgot to update this part 🔢  Now that everything is handled by `disable_cache` it should be fairly simple.